apps/datahub/src/app/home/home-header/home-header.component.html

```diff
@@ -74,4 +74,8 @@
       [style.opacity]="-0.6 + expandRatio * 5"
     ></datahub-navigation-menu>
   </div>
+  <gn-ui-language-switcher
+    *ngIf="showLanguageSwitcher"
+    class="language-switcher absolute top-0 left-0"
```

Comment:
Making the dropdown a little bit smaller and with a bit of spacing looks a bit better IMO
```
class="language-switcher absolute top-2.5 left-2.5 text-[13px]"
```